makefile

```diff
@@ -16,18 +23,21 @@ all: $(OUTFILE)
 open: $(OUTFILE)
 	xdg-open "$(OUTFILE)" &>/dev/null & disown
 
-update: $(WEBSITE)/$(OUTFILE)
+update: $(WEBSITE)/$(DIRECTORY)$(OUTFILE)
 
 clean:
 	rm -f "$(OUTFILE)"
 
 $(OUTFILE): $(SOURCES)
 	pandoc --template="$(TEMPLATE)" --highlight-style="${HIGHLIGHT_STYLE}" --standalone --toc --output "$(OUTFILE)" $(SOURCES)
 
-$(WEBSITE)/$(OUTFILE): $(OUTFILE)
-	cp $(OUTFILE) $(WEBSITE)/documents/
+$(OUTFILE_HTML): $(SOURCES)
+	pandoc --filter="$(FILTER_HTML)" --template="$(TEMPLATE_HTML)" --highlight-style="${HIGHLIGHT_STYLE}" --standalone --toc --output "$(OUTFILE_HTML)" $(SOURCES)
+
+$(WEBSITE)/$(DIRECTORY)$(OUTFILE): $(OUTFILE) $(OUTFILE_HTML)
+	cp "$(OUTFILE)" "$(OUTFILE_HTML)" "$(WEBSITE)/$(DIRECTORY)"
 
```

Comment:
You should test if the website is

1. The only change being added to the commit
2. There were actually changes made
3. You are pushing the correct branch to the correct upstream